File: get-cursors.php
<?php
ini_set('display_errors', 1);
error_reporting(E_ALL);
header('Content-Type: application/json');
$cursorDir = 'cursors/';
if (!is_dir($cursorDir)) {
    echo json_encode(['error' => 'Cursor directory not found']);
    exit;
}
$cursors = array_diff(scandir($cursorDir), array('..', '.'));
if (empty($cursors)) {
    echo json_encode(['error' => 'No cursors found']);
    exit;
}
$cursors = array_map(function($cursor) {
    return $cursor . '.png';
}, $cursors);
echo json_encode($cursors);
?>

File: get-cursors.py
import os
import json

def get_cursors(cursor_dir="cursors"):
  """Возвращает список имен файлов курсоров с расширением .png."""
  try:
    cursors = [f for f in os.listdir(cursor_dir) if not f.startswith('.')]
    if not cursors:
      return json.dumps({'error': 'No cursors found'})
    return json.dumps([f'{f}.png' for f in cursors])
  except FileNotFoundError:
    return json.dumps({'error': 'Cursor directory not found'})

if __name__ == '__main__':
  print(get_cursors())
